### PR TITLE
Add a basic openmp which gpu example

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,10 @@ bin/parallel_kernels_acc: src/parallel_kernels.cpp
 	mkdir -p bin/
 	nvc++ -cuda -acc -Minfo=accel -gpu=lineinfo -o $@ $^
 
+bin/openmp_example: src/openmp_example.cpp
+	mkdir -p bin/
+	nvc++ -Minfo=accel,mp -acc -mp=gpu -gpu=cuda10.2 -o $@ $^
```
Comment (olupton) [inline Makefile]: 10.2 won't be portable to BB5, for example.

File: Makefile [hunk 2]
```diff
@@ -8,6 +8,10 @@ bin/parallel_kernels_acc: src/parallel_kernels.cpp
...
+
 profiles/%.qdrep: bin/%
 	mkdir -p profiles/
 	nsys profile -o $@ -c cudaProfilerApi --kill=none $^
```
Comment (olupton) [inline Makefile:17]: I guess this will never record anything for your new test that doesn't have a CUDA profiler start/stop.

Comment (alkino) [reply Makefile:17]: true

File: Makefile (hunk 1 above)
Comment (alkino) [reply Makefile]: removed

**Review by olupton — APPROVED** [review]

**Pull request merged**